compiler/rustc_typeck/src/check/cast.rs

```diff
@@ -215,6 +236,8 @@ impl<'a, 'tcx> CastCheck<'tcx> {
     }
 
     fn report_cast_error(&self, fcx: &FnCtxt<'a, 'tcx>, e: CastError) {
+        debug!(?e);
```

Comment:
Temporary logging like this is unlikely to help anyone in the future and is better removed to avoid cluttering the code.